pandas/core/sorting.py

```diff
@@ -425,6 +427,10 @@ def safe_sort(values, labels=None, na_sentinel=-1, assume_unique=False):
     assume_unique : bool, default False
         When True, ``values`` are assumed to be unique, which can speed up
         the calculation. Ignored when ``labels`` is None.
+    check_outofbounds : bool, default True
```

Comment:
this is not a bad name but not consistent across pandas, we use verify elsewhere.

Reply:
can you update & add a versionadded tag